content/home/talks.md

```diff
@@ -25,7 +25,7 @@ exclude_selected = false
 
 **[CVPIA SDM](http://cvpia.scienceintegrationteam.com/meetings/)**     
 Conference call     
-September 11, 2019 (10 am - 12 pm)   
+September 25, 2019 (10 am - 12 pm)   
 **[Join Zoom](https://oregonstate.zoom.us/j/3571021423    )**
 Meeting ID: 357 102 1423  
-**[Agenda](http://cvpia.scienceintegrationteam.com/meetings/#agenda )**
\ No newline at end of file
+**[Agenda](http://cvpia.scienceintegrationteam.com/meetings/)**    
```

Comment:
`**[Agenda](http://cvpia.scienceintegrationteam.com/meetings/#agenda )**`
the `#agenda` navigates to the section with the agenda on the meetings page